src/tools/native-client/src/main/java/com/vesoft/client/NativeClient.java

```diff
@@ -59,80 +55,6 @@ public String toString() {
     private static final int VERTEX_SIZE = PARTITION_ID + VERTEX_ID + TAG_ID + TAG_VERSION;
     private static final int EDGE_SIZE = PARTITION_ID + VERTEX_ID + EDGE_TYPE + EDGE_RANKING + VERTEX_ID + EDGE_VERSION;
 
-    private SstFileWriter writer;
-
-    public NativeClient(String path) {
-        EnvOptions env = new EnvOptions();
-        Options options = new Options();
-        options.setCreateIfMissing(true)
-                .setCreateMissingColumnFamilies(true);
-        new NativeClient(path, env, options);
-    }
-
-    public NativeClient(String path, EnvOptions env, Options options) {
-        if (path == null || path.trim().length() == 0) {
-            throw new IllegalArgumentException("File Path should not be null and empty");
-        }
-        writer = new SstFileWriter(env, options);
-        try {
-            writer.open(path);
-        } catch (RocksDBException e) {
-            LOGGER.error("SstFileWriter Open Failed {}", e.getMessage());
-        }
-    }
-
-    public boolean addVertex(String key, Object[] values) {
```

Comment:
why remove this methods ?

Reply:
Cause nobody need it.

It's the `encode` method which is actually need. As a NativeClient' client, Spark job  just doesn't kown how to encode the vertex/edge's key, not how to create a sst file, while the latter case,  would just call Rocksdb Java API directly, not through NativeClient.addXXX methods. So, that make those methods useless.